.syncpackrc

```diff
@@ -1,6 +1,12 @@
 {
+  "formatRepository": false,
```

Comment:
nits

see https://jamiemason.github.io/syncpack/config/format-repository/

Reply:
When `true`,  

```json
  "repository": {
    "type": "git",
    "url": "https://github.com/liam-hq/liam.git"
  }
```  

becomes  

```json
  "repository": "liam-hq/liam"
```  

This is also valid (see [npm docs](https://docs.npmjs.com/cli/v10/configuring-npm/package-json#repository) ), but I thought providing the full URL explicitly would be more convenient.